bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "customapps-medallia-library-FileSaver.js",
+  "version": "0.0.1",
```

Comment:
I agree, but the base project has no versioning, there is a forked project bower-compatible, but I preferred to create the fork from the original one.
So, thoughts? @Jarlakxen @cverges-medallia